llvm/test/CodeGen/AMDGPU/calling-conventions.ll

```diff
@@ -2078,4 +2078,13 @@ entry:
   ret void
 }
 
+; test that i1 argument for amdgpu_cs is working
+define amdgpu_cs void @amdgpu_cs_i1_arg(i1 %arg0) {
+; GCN-LABEL: amdgpu_cs_i1_arg:
```

Comment:
An unused argument doesn't show much useful, should add some sample use so we can see the register used